src/oop/static_modifier/hard/q3/QuestionText.md

```diff
@@ -0,0 +1,32 @@
+# oop-static_modifier-hard-q3
+
+- Question Type: Java question
+
+## CandyStore Class
+
+Your `CandyStore` class must have the following requirements:
+
+- Attributes:
+    - Note that all attributes should not be visible from another class
+    - A _static_ `int` called **gummies**. This represents the number of gummies of the all `CandyStore` objects,
+      and should be set initially to 100 without having to call the constructor.
+    - A `String` called **name**
+
+- Constructors:
+    - A constructor that is accessible from other methods, and accepts a `String` as a parameter to set the **name**
+      attribute of the object.
+
+- Methods:
+    - Note that all methods in this class should be visible from another class
+    - A `award` method which takes an `int` value as input and decrease that value from the **gummies** attribute. Then
```

Comment:
I think we can also add a check here that there are more gummies available than we try to give away.